.github/workflows/ci.yml

```diff
@@ -144,6 +144,10 @@ jobs:
           ${INSTALL_PREFIX}/bin/cadet-cli --version || true
           ${INSTALL_PREFIX}/bin/createLWE
           ${INSTALL_PREFIX}/bin/cadet-cli LWE.h5 || true
+      - name: Run tests
+        run: |
+          cd "${env:BUILD_DIR}\test"
+          .\testRunner.exe [CI]
```

Comment:
In POSIX, there is not `.exe` suffix.

Reply:
yes, i fixed that but it still seems to not find the testrunner ... any ideas?